packages/cli/src/project/operations/file-update.ts

```diff
@@ -27,8 +27,22 @@ export class FileUpdate implements IProjectAttributeOperation {
       throw new Error(`"${filePath}" does not exist`);
     }
 
+    let content: string;
+    if (this._content) {
+      if (typeof this._content === 'function') {
+        // read file content
+        const fileContent = fs.readFileSync(filePath, 'utf8');
+        // call the function with the content
+        content = this._content(fileContent);
+      } else {
+        content = this._content;
+      }
+    } else {
```

Comment:
better to simplify this statement by initializing `content` with an empty string, same with line 38

Reply:
that works. Assigning `content` to `this._content` in line 38 will cause it to have an ambiguous type - either `function` or `string`.